docs/understanding-results.md

```diff
@@ -0,0 +1,248 @@
+
+## Top-level Object
+
+The top-level object is what the lighthouse node module returns and the entirety of the JSON output of the CLI. It contains some metadata about the run and the results in the various subproperties below.
+
+### Properties
+
+| Name | Description |
+| - | - |
+| lighthouseVersion | The version of Lighthouse with which these results were generated. |
+| generatedTime | The ISO-8601 timestamp of when the results were generated. |
+| userAgent | The user agent string of the version of Chrome that was used by Lighthouse. |
+| initialUrl | The URL that was supplied to Lighthouse and initially navigated to. |
+| url | The URL that Lighthouse ended up auditing after redirects were followed. |
+| score | The overall score `0-100`, a weighted average of all category scores. |
+| [audits](#audits) | An object containing the results of the audits. |
+| [runtimeConfig](#runtime-config) | An object containing information about the configuration used by Lighthouse. |
+| [timing](#timing) | An object containing information about how long Lighthouse spent auditing. |
+| [reportCategories](#report-categories) | An array containing the different categories, their scores, and the results of the audits that comprise them. |
+| [reportGroups](#report-groups) | An object containing the display groups of audits for the report. |
+| [artifacts](#artifacts) | *(PROGRAMMATIC USE ONLY)* An object containing gatherer results. |
+
+### Example
+```json
+{
+  "lighthouseVersion": "2.4.0",
+  "generatedTime": "2017-10-05T20:50:54.185Z",
+  "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3233.0 Safari/537.36",
+  "initialUrl": "http://example.com",
+  "url": "https://www.example.com/",
+  "score": 50,
+  "audits": {...},
+  "runtimeConfig": {...},
+  "timing": {...},
+  "reportCategories": [{...}],
+  "reportGroups": {...},
+}
+```
+
+
+<a name="audits"></a>
+## `audits`
+
+An object containing the results of the audits, keyed by their name.
+
+### Audit Properties
+| Name | Type | Description |
+| -- | -- | -- |
+| name  | `string` | The string identifier of the audit in kebab case.  |
+| category | `string` | No longer used. *WARNING: Deprecated, will be removed in Lighthouse 3.0* |
+| description | `string` | The brief description of the audit's successful state. |
```

Comment:
missing failureDescription. see also https://github.com/GoogleChrome/lighthouse/blob/master/docs/architecture.md#auditreport-terminology

Reply:
discussed offline, but not needed since `failureDescription` is just impl detail